### PR TITLE
Use go flags instead of SAJ ENV_VAR

### PR DESCRIPTION
**Pull request by ericbottard**

### CODE REVIEW

File: cmd/function-sidecar.go [hunk 1]
```diff
@@ -27,59 +26,83 @@ import (
...
-	consumerConfig := makeConsumerConfig()
-	consumerConfig.Consumer.Offsets.Initial = sarama.OffsetOldest
+	flag.Parse()
 
-	brokers := []string{saj["spring.cloud.stream.kafka.binder.brokers"].(string)}
-	input := saj["spring.cloud.stream.bindings.input.destination"].(string)
-	output := saj["spring.cloud.stream.bindings.output.destination"]
-	group := saj["spring.cloud.stream.bindings.input.group"].(string)
-	protocol := saj["spring.profiles.active"].(string)
+	input := inputs[0]
```
Comment (markfisher) [inline cmd/function-sidecar.go]: since we are only using the first input and the first output, why are we using a slice?

Comment (ericbottard) [reply cmd/function-sidecar.go]: We could use a single valued var, but if we aim to support multiple inputs/outputs (and I think we said we'd do) I wanted the exposed name of the arg to reflect that _now_, as it's part of the "API" of the sidecar.
You might argue that this is not user exposed though :/

Comment (markfisher) [reply cmd/function-sidecar.go]: can we log a warning if `len > 1` stating that multiple destinations is not yet supported?

Comment (ericbottard) [reply cmd/function-sidecar.go]: We sure can (even though I'm not sure what "warning" really means with the basic go log library). As a matter of fact, I think it's even ok to panic completely

Comment (markfisher) [reply cmd/function-sidecar.go]: sure that would work, and let's create an issue for multiple destinations now if it doesn't exist yet (could even refer to that issue in the log message when we panic)

Comment (ericbottard) [reply cmd/function-sidecar.go]: Created https://github.com/projectriff/riff/issues/184 as this obviously spans more than just the sidecar. will update the code with a ref to that URL

**Comment by markfisher** [comment]
LGTM

**Comment by markfisher** [comment]
squashed and merged as https://github.com/projectriff/function-sidecar/commit/70a50c164449b65f6db9cd9a8792124dbf66a662

**Pull request closed without merging**